src/EPR.RegulatorService.Facade.API/appsettings.json

```diff
@@ -83,8 +83,8 @@
     "AccountCreationUrl": "https://localhost:7854/create-account/invitation/",
     "OrganisationRegistrationSubmissionQueriedId": "cda3cad1-fa77-4282-b39b-7524b232615d",
```

Comment:
shouldn't all these be in secrets?

Reply:
All the other email ID are stored within the app setting file.